ownership-chain/precompile/asset-metadata-extender/contracts/AssetMetadataExtender.sol

```diff
@@ -70,4 +70,22 @@ interface AssetMetadataExtender {
         string calldata uloc,
         uint32 index
     ) external view returns (string memory);
+
+    /// @notice Retrieves the extension of a token's universal location by a claimer
+    /// @param _universalLocation Universal location of the token
+    /// @param _claimer Address of the claimer
+    /// @return Extension of the token's universal location
```

Comment:
remove "token", so it reads: "@return the extension of the universal location by the provided claimer"